framework/src/Volo.Abp.AspNetCore.Mvc/Volo/Abp/AspNetCore/Mvc/ApplicationConfigurations/AbpApplicationConfigurationAppService.cs

```diff
@@ -171,7 +171,7 @@ protected virtual async Task<ApplicationAuthConfigurationDto> GetAuthConfigAsync
 
         foreach (var policyName in policyNames)
         {
-            if (await _defaultAuthorizationPolicyProvider.GetPolicyAsync(policyName) == null && _permissionDefinitionManager.GetOrNull(policyName) != null)
+            if (await _defaultAuthorizationPolicyProvider.GetPolicyAsync(policyName) == null && _permissionDefinitionManager.GetOrNullAsync(policyName) != null)
```

Comment:
`_permissionDefinitionManager.GetOrNullAsync(policyName)` should be awaited.

Reply:
Thanks for finding this. I've fixed.